backend/tracim_backend/lib/core/content.py

```diff
@@ -2490,3 +2503,55 @@ def set_content_namespace(
         )
 
         return content
+
+    def get_content_property(
+        self,
+        property_list: typing.List[QueryableAttribute],
+        content_id: int = None,
+        revision_id: int = None,
+    ) -> typing.Tuple:
+        """
+        Return the specified properties of a revision.
+        - If revision_id is provided, return the properties for that revision.
+        - If only content_id is provided, return the properties of the last revision for that content.
```

Comment:
Is there a use case where we really not to get data without to give revision_id?